src/components/BackButton/index.js

```diff
@@ -0,0 +1,34 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { useHistory } from 'react-router-dom'
+
+import { IconButton } from 'components/Button'
+
+import { ReactComponent as Back } from 'static/icons/back-circle.svg'
+
+import styles from './BackButton.styles'
+
+const goBack = ({ history }) => () => {
+  const { host } = window.location
+  const { referrer } = document
+
+  if (history.length > 1 && referrer && referrer.includes(host)) {
```

Comment:
This prevents us from going back to a place that is offsite. eg: if you link from an email directly to a page within our site and click back it would normally take you back to the email. Instead we'll send users back to the homepage.

Reply:
Nice!